src/util/AppContextUtil/Shogun2AppContextUtil.tsx

```diff
@@ -291,14 +293,29 @@ class Shogun2AppContextUtil extends BaseAppContextUtil implements AppContextUtil
     const layerSource = new OlTileWMS({
       url: url,
       attributions: attribution,
-      tileGrid: tileGrid,
+      tileGrid: baseTileGrid,
+      projection: 'EPSG:3857',
```

Comment:
The projection should be the mapView's projection by default. But is is always null. If specified here, the code works - but this cannot be the final solution. Ideas?

Reply:
Any chance to get the value from the given crs defined for the current application?

Reply:
Maybe to store it as private property within the class? see new changes. Not sure though

Reply:
This is fine for me.

But please check lint warnings.

Reply:
yup, thanks for review